### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by CallToPower**
Added ui-notifications + a factory as a wrapper for the translated notifications.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {
   "name": "mindsmash-ui",
-  "description": "A ui kid by mindsmash",
-  "main": ["dist/mindsmash-ui.js", "dist/stylesheets/mindsmash-ui.scss", "dist/stylesheets/_variables.scss"],
+  "description": "A ui kit by mindsmash",
```
Comment (badabam) [inline bower.json:3]: Haha, kid! Thanks!

File: source/docs/index.html [hunk 2]
```diff
@@ -224,13 +236,14 @@ <h3 class="panel-title-small">Welcome to our new office</h3>
 <script src="../../bower_components/holderjs/holder.js"></script>
 <script src="../../bower_components/highlightjs/highlight.pack.js"></script>
 <script src="../../bower_components/angular-translate/angular-translate.js"></script>
+<script src="../../bower_components/angular-ui-notification/dist/angular-ui-notification.min.js"></script>
```
Comment (badabam) [inline source/docs/index.html:240]: I usually choose the un-minified version in here to make the source more readable, if necessary. It's fine though.

File: bower.json (hunk 1 above)
Comment (CallToPower) [inline bower.json:3]: Was kind of funny ;)

**Comment by badabam** [comment]
Very nice, ship it! (please squash-merge)

**Pull request merged**